misc/style/run-clang-tidy.py

```diff
@@ -47,15 +48,29 @@ def check_search_code_with_clang_tidy():
     # categories instead of deleting them to see which additional checks
     # we could activate.
     checks = [
-        # Enable with CheckTriviallyCopyableMove=0 when we require
-        # clang-tidy >= 6.0 (see issue856).
```

Comment:
Did our required clang-tidy version change, or why is the comment no longer applicable?

Reply:
We require `run-clang-tidy-12` since a while, so I assume the comment was outdated.

Reply:
I just moved the comment down in the file.

Reply:
If it's outdated, it would be better to remove it.

Reply:
It's not outdated. See the linked issue.